src/pages/task/taskTabs/TestsTable.tsx

```diff
@@ -113,39 +83,89 @@ export const TestsTable: React.FC<TestsTableProps> = ({ task }) => {
   });
   usePolling({ startPolling, stopPolling, refetch });
 
-  // update url query params when user event triggers change
-  const tableChangeHandler: TableOnChange<TestResult> = (...[, , sorter]) => {
-    const { columnKey, order } = Array.isArray(sorter) ? sorter[0] : sorter;
-    let queryParams = {
-      [RequiredQueryParams.Category]: undefined,
-      [RequiredQueryParams.Sort]: undefined,
-      [RequiredQueryParams.Page]: "0",
-    };
-    if (order !== undefined) {
-      queryParams = {
-        ...queryParams,
-        [RequiredQueryParams.Category]: `${columnKey}`,
-        [RequiredQueryParams.Sort]:
-          order === "ascend" ? SortDirection.Asc : SortDirection.Desc,
-      };
-    }
-    updateQueryParams(queryParams);
+  const clearQueryParams = () => {
```

Comment:
I noticed some flaky behavior on this button where sometimes the filters didn't actually clear (this is on beta, with only original PR changes applied). I couldn't really tell why this happens though... 🤔

https://github.com/evergreen-ci/spruce/assets/47064971/1d9feb6e-8819-407a-8c32-6a15561ef4f5